test/PrintAsObjC/extensions.swift

```diff
@@ -95,8 +118,7 @@ extension GenericClass {
 class NotObjC {}
 extension NotObjC {}
 
-// CHECK-LABEL: @interface NSObject (SWIFT_EXTENSION(extensions))
-// CHECK-NEXT: @end
+// NEGATIVE-NOT: @interface NSObject (SWIFT_EXTENSION(extensions))
 // NEGATIVE-NOT: @interface NSObject{{$}}
 // NEGATIVE-NOT: @class NSObject
 extension NSObject {}
```

Comment:
This part is supposed to be a minimal test that we don't try to *define* an imported class. Can you put a member in instead, like you did with A2?